src/tools/initSchemas.ts

```diff
@@ -0,0 +1,11 @@
+import './scriptInit'
+
+import { ToolsUtils } from 'tools/utils/toolsUtils'
+
+import { DBService } from 'server/db/service/dbService'
+
+const exec = async (): Promise<void> => {
```

Comment:
let's move all db tools under src/tools/db (including importTables and exportTables).
what do you think ?